<launch>

  <node name="leftarm_setup" pkg="shell_game" type="leftarm_setup.py" />

  <node name="image_view" pkg="image_view" type="image_view" >
    <remap from="/image" to="/cameras/left_hand_camera/image" />
  </node>

</launch>
